aspnet/tutorials/nano-server.rst

```diff
@@ -40,55 +40,75 @@ Open an elevated PowerShell window to add your remote Nano Server instance to yo
+.. code:: ps1
 
+  New-NetFirewallRule -Name "AspNet5 IIS" -DisplayName "Allow HTTP on TCP/8000" -Protocol TCP -LocalPort 8000 -Action Allow -Enabled True
 
 Installing IIS
 --------------
 
 Add the ``NanoServerPackage`` provider from the PowerShell gallery. Once the provider is installed and imported, you can install Windows packages.
 
+Run the following commands in the PowerShell session that was created earlier:
+
 .. code:: ps1
 
   Install-PackageProvider NanoServerPackage
   Import-PackageProvider NanoServerPackage
   Install-NanoServerPackage -Name Microsoft-NanoServer-IIS-Package
 
+To quickly verify if IIS is setup correctly, you can visit the url ``http://<nanoserver-ip-address>/`` and should see a welcome page. When IIS is installed, by default a web site called ``Default Web Site`` listening on port 80 is created.
 
-Installing the ASP.NET Core Module
-----------------------------------
+Installing the ASP.NET Core Module (ANCM)
+-----------------------------------------
 
-The ASP.NET Core Module is an IIS 7.5+ module which is responsible for process management of ASP.NET Core HTTP listeners and to proxy requests to processes that it manages. Currently, installing ASP.NET Core Module requires the following script:
+The ASP.NET Core Module is an IIS 7.5+ module which is responsible for process management of ASP.NET Core HTTP listeners and to proxy requests to processes that it manages. At the moment, the process to install the ASP.NET Core Module for IIS is manual. You will need to install the version of the `.NET Core Windows Server Hosting bundle <https://dot.net/>`__ on a regular (not Nano) machine. After installing the bundle on a regular machine, you will need to copy the following files to the file share that we created earlier.
 
 .. code:: ps1
 
-  copy <update-this>\aspnetcore_schema.xml C:\windows\system32\inetsrv\config\schema
-  copy <update-this>\aspnetcore.dll C:\windows\system32\inetsrv
+  copy C:\windows\system32\inetsrv\aspnetcore.dll ``\\<nanoserver-ip-address>\AspNetCoreSampleForNano``
+  copy C:\windows\system32\inetsrv\config\schema\aspnetcore_schema.xml ``\\<nanoserver-ip-address>\AspNetCoreSampleForNano``
 
-  # Backup existing applicationHost.config
+
+Run the following script in the remote session:
+
+.. code:: ps1
+
+  copy \\<nanoserver-ip-address>\AspNetCoreSampleForNano\aspnetcore_schema.xml C:\windows\system32\inetsrv\config\schema
+  copy \\<nanoserver-ip-address>\AspNetCoreSampleForNano\aspnetcore.dll C:\windows\system32\inetsrv
+
+  # Backup existing applicationHost.config before installing ANCM
   copy C:\Windows\System32\inetsrv\config\applicationHost.config C:\Windows\System32\inetsrv\config\applicationHost_BeforeInstallingANCM.config
 
   Import-Module IISAdministration
```

Comment:
Can you move this inline powershell script back into this file https://github.com/aspnet/Docs/blob/master/aspnet/tutorials/nano-server/enable-ancm.ps1